### PR TITLE
feat: validate makecomapp.json by json schema

### PR DESCRIPTION
**Pull request by fuka-michal**
## Change Log

- Added in-editor validation of `makecomapp.json` project file.

### CODE REVIEW

**Review by ma-zal — APPROVED** [review]

File: src/extension.test.ts [hunk 1]
```diff
@@ -126,6 +126,18 @@ suite('App online file edit validations', () => {
 			problematicContent: '{"invalidProperty":"someValue"}',
 			expectedProblemMessage: 'Incorrect type. Expected "array".',
 		},
+		{
+			filename: 'makecomapp.json',
```
Comment (fuka-michal) [inline src/extension.test.ts:130]: wrong file name? makecomapp.schema.json

File: package.json [hunk 2]
```diff
@@ -132,6 +132,12 @@
 					"groups.json"
 				],
 				"url": "./syntaxes/imljson/schemas/groups.json"
+			},
+			{
+				"fileMatch": [
+					"makecomapp.json"
```
Comment (fuka-michal) [inline package.json:138]: wrong file name? makecomapp.types.ts

File: src/extension.test.ts (hunk 1 above)
Comment (ma-zal) [reply src/extension.test.ts:130]: It is correct. It defines the simulated filename, under which the testing file is opened in editor.

File: package.json (hunk 2 above)
Comment (ma-zal) [reply package.json:138]: It is correct. It defines the simulated file name. It means the testing file opened in editor will have this name.

**Comment by ma-zal** [comment]
Ready to merge.
Note: CI `PR / Validate PR` is being ignored, because not configured correctly.

**Pull request merged**